styles.css

```diff
@@ -509,6 +509,42 @@ ul {
   display: none;
 }
 
+.desktop-nav a {
+  height: 20px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  margin-right: 40px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.topbar.visible {
+  position: relative;
+  background: #2b2928;
+}
+
+.topbar.visible .desktop-nav {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 72px;
+  left: 0;
+  float: none;
+  height: auto;
+  background: #3c3a39;
+  width: 100%;
+  align-items: center;
+}
```

Comment:
The style doesn't quite match that of the template design.

<img width="229" alt="Screen Shot 2022-07-18 at 16 29 47" src="https://user-images.githubusercontent.com/82426682/179620729-1c5aa29f-5a31-47c5-b258-8703ba14a3a6.png">

<img width="316" alt="Screen Shot 2022-07-18 at 16 30 32" src="https://user-images.githubusercontent.com/82426682/179620848-f6fa3c47-f6a0-4300-8691-5a0b0798d29f.png">

Make sure the menu covers the entirety of the screen in both axis and make sure the style of the menu matches that of the template design once you enlarge it.

<img width="836" alt="Screen Shot 2022-07-18 at 16 31 56" src="https://user-images.githubusercontent.com/82426682/179621084-d70afa63-b306-4abf-9795-ec1fd2dc4c36.png">